src/components/PopUps/PopUpMessageBox.tsx

```diff
@@ -113,6 +114,16 @@ const PopUpMessageBox = ({message}: PopUpMessageProps) => {
                 <Typography level={"body-sm"}>
                     {messageStr}
                 </Typography>
+                {button && (
+                    <Button
+                        className={"pop-up-message-box-callback-button"}
+                        color={color}
+                        variant={"soft"}
```

Comment:
Since this is the one-and-only-one action button in this pop-up, how about
```suggestion
                        variant={"solid"}
```
(think about we will add a secondary action button in the future)